Tasks/DownloadSecureFileV1/predownloadsecurefile.ts

```diff
@@ -27,6 +28,15 @@ async function run() {
         if (tl.exist(secureFilePath)) {
             // set the secure file output variable.
             tl.setVariable('secureFilePath', secureFilePath);
+
+            var errorFileContent = fs.readFileSync(secureFilePath).toString();
```

Comment:
Is this expected that error is being written inside of downloaded file - instead of throwing as exception?
Should we fix it somewhere in common package instead or other inner logic which writes exception to the file (see original issue)?